### PR TITLE
Update NuGet.config in HttpStress

### PR DESCRIPTION
**Pull request by hoyosjs**
We can't use nuget.org in this manner to comply with security guidance.

### CODE REVIEW

**Review by kunalspathak — APPROVED** [review]
LGTM. Thanks!

**Review by BruceForstall — APPROVED** [review]
Thank you!

**Review by safern — APPROVED** [review]

**Comment by hoyosjs** [comment]
Once the HttpStress tests build I'll merge this to unblock the JIT rolling builds.

File: src/libraries/System.Net.Http/tests/StressTests/HttpStress/NuGet.config
Comment (ManickaP) [inline src/libraries/System.Net.Http/tests/StressTests/HttpStress/NuGet.config:4]: Why do you clear this? We need what's in repository wide nuget.config

File: src/libraries/System.Net.Http/tests/StressTests/HttpStress/NuGet.config
Comment (ManickaP) [inline src/libraries/System.Net.Http/tests/StressTests/HttpStress/NuGet.config:5]: Will all the packages we use be there? Could you run the stress pipeline on this PR? It's not part of inner loop.

**Review by ManickaP — CHANGES_REQUESTED** [review]
I think this will break our stress pipeline. Could we run it before we merge?

File: src/libraries/System.Net.Http/tests/StressTests/HttpStress/NuGet.config
Comment (hoyosjs) [reply src/libraries/System.Net.Http/tests/StressTests/HttpStress/NuGet.config:4]: If this is true, then you should not have this file at all. We don't allow nuget.org. Let me delete this and we'll see the fallout. FYI, this already broke other partners within the runtime repo.

Comment (ManickaP) [reply src/libraries/System.Net.Http/tests/StressTests/HttpStress/NuGet.config:4]: Let's try. BTW, `/azp run runtime-libraries stress-http` is the magic to run the pipeline.

Comment (ManickaP) [reply src/libraries/System.Net.Http/tests/StressTests/HttpStress/NuGet.config]: One more thing, can we import packages from nuget.org to `dotnet-public` if they are missing? What's the process for that?

**Review by safern — APPROVED** [review]

**Review by ManickaP — APPROVED** [review]
The [http stress pipeline run](https://dev.azure.com/dnceng/public/_build/results?buildId=1233075&view=results) for the last change.
I don't want to wait for it, so approving, thanks.

**Comment by ManickaP** [comment]
The pipeline looks good :shipit:

**Pull request merged**
